pkg/kubectl/cmd/annotate/annotate.go

```diff
@@ -41,8 +41,8 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/util/templates"
 )
 
-// AnnotateOptions have the data required to perform the annotate operation
```

Comment:
leave this naming as before

Reply:
Okay.

But, apart from the above-mentioned comment, is there any other note on this specific requirement? I am asking this because the `XYZOptions` lint error is present in almost every package. This should be noted down somewhere so that in future such changes are never proposed.

Reply:
Also, does the same logic hold for types like `DeleteFlags`?

Reply:
ping @prksu

Reply:
Hi @palnabarun

you can see in the kubectl [Command implementation conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-cli/kubectl-conventions.md#command-implementation-conventions)